src/common.c

```diff
@@ -1602,19 +1602,22 @@ static void ofi_pollfds_do_del(struct ofi_pollfds *pfds,
 	if (ofi_poll_fairness && ctx->hot_index >= 0)
 		ofi_pollfds_coolfd(pfds, item->fd);
 
+	last_pfd = &pfds->fds[pfds->nfds - 1];
+	last_ctx = ofi_pollfds_get_ctx(pfds, last_pfd->fd);
```

Comment:
Btw, I used swap_ctx rather than last_ctx, because this may not be the last context.

Reply:
Doesn't it have to be the last [used] context by definition? `[pfds->nfds-1]`
I left the 'used' out of the rename. I can switch it back to swap if you prefer.